lib/faker/default/tea.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Faker
+  class Tea < Base
+    flexible :tea
+
+    class << self
+      ##
+      # Produces a random variety or blend of tea
+      #
+      # @param type [String, nil] the type of tea to query for (valid types: 'Black', 'Green', 'Oolong', 'White', and 'Herbal')
+      # @return [String] a variety of tea
+      #
+      # @example
+      #   Faker::Tea.variety
+      #     #=> "Earl Grey"
+      #
+      # @example
+      #   Faker::Tea.variety(type: 'Green')
+      #     #=> "Jasmine"
+      def variety(type: nil)
```

Comment:
Can you add `@faker.version next`?

```suggestion
      # @example
      #   Faker::Tea.variety(type: 'Green')
      #     #=> "Jasmine"
      #
      # @faker.version next
      def variety(type: nil)
```